typings/index.d.ts

```diff
@@ -1046,7 +1046,7 @@ declare module 'discord.js' {
 
 	export class User extends PartialTextBasedChannel(Base) {
 		constructor(client: Client, data: object);
-		public avatar: string;
+		public avatar?: string;
```

Comment:
`avatar` is nullable, not optional.

Reply:
I used the `?` there, because `avatar?` imples `avatar | undefined`. 
User.avatar doesn't seem to be explicitly nullable, since I assumed that this will result to `undefined` instead of `null`, due to https://github.com/discordjs/discord.js/blob/9b2bf03ff6041148a0c6c6351a2bd74c44b4166a/src/structures/User.js#L54
So, when a User doesn't have an avatar, the property isn't even set, unless I misunderstood this?